QUBEKit/dihedrals.py

```diff
@@ -66,6 +67,14 @@ def __init__(self, molecule, constraints_made=None):
         # working dir
         self.home = os.getcwd()
 
+    def is_short_cc_bond(self, bond):
```

Comment:
maybe more of a reason to have a bond class struct in the ligand that already has this info? either way this should be in ligand?